day_7/ceasar_cipher/ceasar_cipher.rb

```diff
@@ -0,0 +1,49 @@
+# Thanks to matugm @ https://gist.github.com/matugm/db363c7131e6af27716c
+# for this method of running a ceasar cipher.
+
```

Comment:
Same thing here... Inspiration is ok, but we don't want you copy/pasting. The end product is often less important than struggling through a problem to deeply learn what it takes to solve it